bot/cogs/help_channels.py

```diff
@@ -463,14 +476,14 @@ def is_dormant_message(message: t.Optional[discord.Message]) -> bool:
             category=self.in_use_category,
             sync_permissions=True,
             topic=IN_USE_TOPIC,
-            position=10000,
```

Comment:
I think you messed up a merge somewhere because this position was never supposed to exist in the first place.

Reply:
Fixed